src/components/Favorites/Favorite.component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { List } from 'immutable';
+import propTypes from 'prop-types';
+
+import {
+  FavoritesWrapper,
+  FavoritesTitle,
+  FavoritesSongListWrapper,
+} from './Favorites.style';
+
+import LinkToSongContainer from '../Files/LinkToSong';
+
+const FavoriteComponent = ({ favorites }) => (
```

Comment:
Let's use Grid elements as possible instead of custom wrappers.

https://material-ui.com/components/grid/